packages/cms-cli/commands/secrets/updateSecret.js

```diff
@@ -40,6 +41,8 @@ exports.handler = async options => {
   trackCommandUsage('secrets-update', {}, portalId);
 
   try {
+    const { secretValue } = await secretValuePrompt();
```

Comment:
Are we doing this for a security reason?  Wouldn't the secret still be copied to your clipboard if you are passing it into inquirer?

Reply:
Copied to clipboard, yes. But my prompting for the value like this we can keep the value out of bash `history`.